engine/etcdng/etcd.go

```diff
@@ -32,14 +35,17 @@ type Options struct {
 	Box             *secret.Box
 }
 
-func New(nodes []string, etcdKey string, registry *plugin.Registry, options Options) (engine.Engine, error) {
+func New(nodes []string, etcdKey, etcdCaFile, etcdCertFile, etcdKeyFile string, registry *plugin.Registry, options Options) (engine.Engine, error) {
```

Comment:
Thanks for the response, I will address this as soon as possible, but maybe it will take the next weekend.